src/main/kotlin/io/snyk/plugin/services/SnykAnalyticsService.kt

```diff
@@ -145,6 +147,22 @@ class SnykAnalyticsService : Disposable {
         }
     }
 
+    fun logQuickFixIsDisplayed(event: QuickFixIsDisplayed) {
+        if (!settings.usageAnalyticsEnabled) return
+
+        catchAll(log, "quickFixIsDisplayed") {
+            itly.logQuickFixIsDisplayed(userId, event)
+        }
+    }
+
+    fun logQuickFixIsTriggered(event: QuickFixIsTriggered) {
+        if (!settings.usageAnalyticsEnabled) return
+
+        catchAll(log, "quickFixIsDisplayed") {
```

Comment:
should be `logQuickFixIsTriggered`
PS was confused not to see any logQuickFixIsTriggered in the logs...

Reply:
Damn copy & paste ;)